packages/Kokkos/src/DTK_PointCloud_def.hpp

```diff
@@ -105,8 +104,8 @@ class LocalBoundingBoxFunctor
     {
         for ( size_t d = 0; d < 3; ++d )
         {
-            dst[d] = std::numeric_limits<SC>::max();
-            dst[d + 3] = -std::numeric_limits<SC>::max();
+            dst[d] = KokkosHelpers::numericLimitsMax<SC>();
+            dst[d + 3] = -KokkosHelpers::numericLimitsMax<SC>();
```

Comment:
The less code we have to maintain ourselves, the better. So I think that we should use the traits from Tpetra. Maybe we could let the kokkos guys that it's a feature that we are interested in. This way they may move it into kokkos (if Tpetra needs it and we need it too, there is a high chance that other people need it too).